railties/lib/rails/source_annotation_extractor.rb

```diff
@@ -29,6 +29,16 @@ def self.register_directories(*dirs)
         directories.push(*dirs)
       end
 
+      def self.tags
```

Comment:
Question: Does this not need to be documented(a small comment above the method name)?
Or do we not do it for methods who have self-explanatory names?

Reply:
I chose to stay consistent with the rest of the code (neither `self.directories` nor `self.extensions` have comments above). Also I think this really does not need any explanation for we would just end up explaining memoization.